src/plugins/guided_onboarding/public/constants/guides_config/observability.ts

```diff
@@ -33,7 +33,8 @@ export const observabilityConfig: GuideConfig = {
       integration: 'kubernetes',
       descriptionList: [
         i18n.translate('guidedOnboarding.observabilityGuide.addDataStep.description', {
-          defaultMessage: 'Start by adding your data by setting up the Kubernetes integration.',
+          defaultMessage:
+            'Start by adding your data by setting up the Kubernetes integration. For the best experience, deploy kube-state-metrics in Kubernetes.',
```

Comment:
Based on the conversation today, my understanding is that this step is not optional and is needed for this to work. If that's correct, I would suggest that we change to bullet points and tweak the wording.

- Deploy [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics)<jumpout icon> service to your Kubernetes.
- Add the Elastic Kubernetes integration.


@osmanis can you confirm the proper order? I'm not sure which one the user should do first. Also, is this link the correct place to send folks? It seemed the most relevant.

Reply:
I think this would be the correct order. The dashboard itself will still load without kube-state-metrics, but many visualizations will not be available.

Reply:
@kellyemurphy I updated the copy per your suggestion and also added a new screenshot to the PR description. Can you take another look?